core/ir/opnd_api.h

```diff
@@ -2009,6 +2008,18 @@ DR_API
 opnd_t
 opnd_create_immed_int(ptr_int_t i, opnd_size_t data_size);
 
+#ifdef RISCV64
+DR_API
+/**
+ * Returns a signed immediate integer operand with value \p i and size
+ * \p data_size; \p data_size must be a OPSZ_ constant.
+ *
+ * The integer will be formatted as a decimal when disassemble.
```

Comment:
grammar: disassembled

Reply:
Fixed in https://github.com/DynamoRIO/dynamorio/pull/6095/commits/4dc5fb5ebcab5db0f70080a3ebd3f4a404457d1f.